frontend/src/features/permits/helpers/getDefaultApplicationFormData.ts

```diff
@@ -184,7 +182,6 @@ export const getDefaultValues = (
   );
 
   return {
-    companyId: +getDefaultRequiredVal(0, companyId),
```

Comment:
Do we still need companyId field for applications/permits? Because this line here simply populates that field...

Reply:
ie. do we still need it for create/update application request data?